From 4b9801d9bbe535fd6719933b96278915573e3595 Mon Sep 17 00:00:00 2001
From: Maximilian Bosch <maximilian@mbosch.me>
Date: Sun, 10 Mar 2019 16:42:21 +0100
Subject: [PATCH] Don't run flake8 checks during the build

If the code simply violates their code style, the Nix package shouldn't fail.
---
 pytest.ini | 2 +-
 setup.cfg  | 1 -
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/pytest.ini b/pytest.ini
index 9b3c1ec..a5189c1 100644
--- a/pytest.ini
+++ b/pytest.ini
@@ -1,6 +1,6 @@
 [pytest]
 norecursedirs=dist build .tox .eggs
-addopts=--doctest-modules --flake8
+addopts=--doctest-modules
 doctest_optionflags=ALLOW_UNICODE ELLIPSIS
 filterwarnings=
 	ignore:Possible nested set::pycodestyle:113
diff --git a/setup.cfg b/setup.cfg
index 3e7bbed..5cac7a2 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -29,7 +29,6 @@ setup_requires = setuptools_scm >= 1.15.0
 testing = 
 	pytest >= 3.5, !=3.7.3
 	pytest-checkdocs
-	pytest-flake8
 docs = 
 	sphinx
 	jaraco.packaging >= 3.2
-- 
2.18.1

